lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
Hey, should this always be the case? Looking through the rules, I'm not seeing any other instances of `additionalProperties: false with this pattern. Should I fix those up too? Some examples:
* https://github.com/eslint/eslint/blob/c67632259783c9630831bce5f272eb6160ee086c/lib/rules/computed-property-spacing.js#L25
* https://github.com/eslint/eslint/blob/c67632259783c9630831bce5f272eb6160ee086c/lib/rules/computed-property-spacing.js#L25
* https://github.com/eslint/eslint/blob/c67632259783c9630831bce5f272eb6160ee086c/lib/rules/space-in-parens.js#L25
* https://github.com/eslint/eslint/blob/c67632259783c9630831bce5f272eb6160ee086c/lib/rules/array-bracket-spacing.js#L23